src/Task/Deptrac.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Runner\TaskResultInterface;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Deptrac task
+ */
+class Deptrac extends AbstractExternalTask
+{
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return 'deptrac';
+    }
+
+    /**
+     * @return OptionsResolver
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults([
+            'formatter_graphviz' => 0,
+            'formatter_graphviz_display' => false,
+            'formatter_graphviz_dump_image' => null,
+            'formatter_graphviz_dump_dot' => null,
+            'formatter_graphviz_dump_html' => null,
+        ]);
+
+        $resolver->addAllowedTypes('formatter_graphviz', ['int']);
```

Comment:
Updated this option so it allows boolean configuration. In deptrac itself this option is an integer though.